controller/src/main/java/org/apache/rocketmq/controller/ControllerStartup.java

```diff
@@ -16,12 +16,6 @@
  */
 package org.apache.rocketmq.controller;
 
-import java.io.BufferedInputStream;
```

Comment:
It's better to only format the code you change or add. It may cause confusion for reviewers if formatting the 
irrelevant file

Reply:
I think It's hard to find your real changes in the file diff if you format the whole file

Reply:
I will rollback it.

Reply:
Thank you :)